src/stuf/tests/test_model.py

```diff
@@ -1,8 +1,23 @@
-from django.test import TestCase
+from uuid import uuid4
 
-from stuf.tests.factories import SoapServiceFactory
+from django.test import SimpleTestCase, TestCase
+
+from stuf.tests.factories import SoapServiceFactory, StufServiceFactory
 
 
 class SoapServiceTest(TestCase):
     def test_factory(self):
         SoapServiceFactory.create()
+
+
+class StufServiceTests(SimpleTestCase):
+    def test_str_display(self):
+        service = StufServiceFactory.build(soap_service__label="service label")
+
+        self.assertEqual(str(service), "service label")
+
+    def test_get_endpoint_bogus_type(self):
+        service = StufServiceFactory.build()
+
+        with self.assertRaises(ValueError):
+            service.get_endpoint(str(uuid4()))
```

Comment:
Is there a chance of 1 in 2 ** 128 this fails?

Cake if this ever fails in CI!